kavm/src/kavm/__main__.py

```diff
@@ -0,0 +1,179 @@
+"""Command-line interface for KAVM"""
+
+import os
+from argparse import ArgumentParser
+from pathlib import Path
+from subprocess import CalledProcessError
+
+from pyk.cli_utils import dir_path
+
+from .kavm import KAVM
+
+
+def main() -> None:
+    parser = create_argument_parser()
+    args = parser.parse_args()
+
+    if not args.definition_dir:
+        env_definition_dir = os.environ.get('KAVM_DEFINITION_DIR')
+        if env_definition_dir:
+            args.definition_dir = Path(env_definition_dir)
+        else:
+            raise RuntimeError('Cannot find KAVM definition, plese specify either --definition or KAVM_DEFINITION_DIR')
+
+    if args.command == 'kompile':
```

Comment:
Consider extracting a function for each of these commands. (You can check `kevm-pyk.__main__` for an [example](https://github.com/runtimeverification/evm-semantics/blob/e7a58e8e83883a6667ac0d798bd069b0321f16b4/kevm-pyk/src/kevm_pyk/__main__.py#L48).)